sdk/template/Azure.Template/api/Azure.Template.netstandard2.0.cs

```diff
@@ -1,35 +1,39 @@
-namespace Azure.Template
+namespace Azure.Template.Models
 {
-    public partial class TemplateClient
+    public abstract partial class BaseClass
     {
-        protected TemplateClient() { }
-        public TemplateClient(string vaultBaseUrl, Azure.Core.TokenCredential credential) { }
-        public TemplateClient(string vaultBaseUrl, Azure.Core.TokenCredential credential, Azure.Template.TemplateClientOptions options) { }
-        public virtual Azure.Core.Pipeline.HttpPipeline Pipeline { get { throw null; } }
-        public virtual Azure.Response GetSecret(string secretName, Azure.RequestContext context = null) { throw null; }
-        public virtual System.Threading.Tasks.Task<Azure.Response> GetSecretAsync(string secretName, Azure.RequestContext context = null) { throw null; }
-        public virtual Azure.Response<Azure.Template.Models.SecretBundle> GetSecretValue(string secretName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
-        public virtual System.Threading.Tasks.Task<Azure.Response<Azure.Template.Models.SecretBundle>> GetSecretValueAsync(string secretName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        internal BaseClass() { }
+        public string BaseClassProperty { get { throw null; } set { } }
     }
-    public partial class TemplateClientOptions : Azure.Core.ClientOptions
+    public abstract partial class BaseClassWithDiscriminator : Azure.Template.Models.BaseClass
```

Comment:
Why does this class not actually have the discriminator property?

Reply:
It does - it's just internal, so you don't see it in the public API.